h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,48 @@
+###*
+# @ngdoc directive
+# @name excerpt
+# @restrict AE
+# @description Checks to see if text is overflowing its container.
+# If so, it prepends/appends expanders/collapsers. Note, to work
+# the element needs a max height.
+###
+module.exports = ->
+  link: (scope, elem, attr, ctrl) ->
+    scope.collapsed = true
+
```

Comment:
Would suggest adding a check here that sets `scope.enabled` to true if it's undefined.